datastore/google/cloud/datastore/client.py

```diff
@@ -74,7 +75,7 @@ def _determine_default_project(project=None):
         project = _get_gcd_project()
 
     if project is None:
-        project = _base_default_project(project=project)
+        project = _default_project(project=project)
```

Comment:
I'm pretty unclear on why this (and the import) were changed at all. They have nothing to do with the PR AFAICT.

Reply:
I think I was thinking I could get the import line to just under the 80 char limit.  But CircleCI was 79 chars.  I can change it back if you want.

Reply:
Please change it back. In general, a PR should **only** contain changes relevant to one thing being changed. (We break this from time to time of course, but should try very hard not to.)